src/screens/profile/screen/profileScreen.js

```diff
@@ -206,6 +206,7 @@ class ProfileScreen extends PureComponent {
                 tag={username}
                 key={username}
                 handleOnScroll={this._handleOnScroll}
+                hideReblogOption
```

Comment:
if the variable include boolean. lets change the name as `isHideReblogOption` or `isReblogOptionHide` its seems more valuable isn't it?